src/app/ui/learn/LearnPageArticlesSection/LearnPageArticlesSection.tsx

```
@@ -0,0 +1,24 @@
+const LearnPageArticlesSectionClient = dynamic(
+  () =>
+    import('./LearnPageArticlesSectionClient').then(
+      (mod) => mod.LearnPageArticlesSectionClient,
+    ),
+  {
+    ssr: false,
+    loading: LearnPageArticlesSectionSkeleton,
+  },
+);
+
+interface LearnPageArticlesSectionProps {}
```

Biome (2.4.7)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)